decentraland/unity-explorer
# fix: empty raycast not partitioning (bird scene)

## What does this PR change?

Fixes #1081 

- Adds raycast to entites that just had a transform
- Fix for the HybridSceneApproach. Some main files are named `game.js`, so we need the main file reference

## How to test the changes?

Recomendation: In Debug Menu, in Landscape setting, move the load raidus to 1.

Go to Goerli
Go to the birds scene (74,-2). You should be able to play as in explorer. Birds should appear and fly away as you run through the scene

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
